provisioning-workflows/IIB/IIBv9/jcl/CRENWDR.jcl

```diff
@@ -0,0 +1,10 @@
+//S1     EXEC USSBATCH,SU=Y
```

Comment:
USSBATCH is not a standard utility available on z/OS, maybe we should find a more standard approach.

Reply:
USSBATCH is now supplied with workflow.